libraries/c_sdk_v4/integration-test/core_mqtt_system_test.c

```diff
@@ -0,0 +1,1635 @@
+/*
+ * AWS IoT Device SDK for Embedded C V202009.00
```

Comment:
This License header is for CSDK. It may be a gray area to decide what is the correct header to use here. I think FreeRTOS header would be good.